components/layout_2020/flow/mod.rs

```diff
@@ -227,15 +227,12 @@ pub(crate) struct CollapsibleWithParentStartMargin(bool);
 #[derive(Debug)]
 pub(crate) struct OutsideMarker {
     pub marker_style: Arc<ComputedValues>,
+    pub list_item_style: Arc<ComputedValues>,
```

Comment:
Is there a case where this field is different from `self.base.style`?

Reply:
Ah nevermind, we'll need to distinguish between the style of the marker pseudo and the list item itself in the future. Makes sense.

Reply:
I still don't get this. Previously `marker_style` was the marker style and `base.style` was the list item style.

Now the list item style is `list_item_style`, and the marker style is both in `marker_style` and `base.style`? Why not remove `marker_style`?